src/DDTrace/Tracer.php

```diff
@@ -120,7 +120,7 @@ public function __construct(Transport $transport = null, array $propagators = nu
 
     public function limited()
     {
-        return dd_trace_tracer_is_limited();
+        return true;
```

Comment:
Why returning `true` is preferable compared to returning `\dd_trace_tracer_is_limited();`? Since in manual tracing users actually use a tracer instance, how would their manual instrumentation be impacted?

Reply:
Felt most sensible when looking at the NullTracer, i.e. default?

Reply:
We just need to remove the `dd_trace_tracer_is_limited()` calls from the integrations, but this is part of the userland instrumentation API so we should keep the call here.

Even though limited checks are being checked internally for the sandbox API (which is what our integrations use), customers who have custom instrumentations using the userland `DDTrace\Tracer` API will still need the option of doing a limited check using `Tracer::limited()`.